packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1756,12 +1756,9 @@ export class AccountHandler {
     this.log.begin('Account.getCredentialsStatus', request);
 
     const email = (request.payload as any).email;
-    await this.customs.check(request, email, 'getCredentialsStatus');
-    const { accountRecord } = await this.signinUtils.checkCustomsAndLoadAccount(
-      request,
-      email,
-      true
-    );
+    await this.customs.checkIpOnly(request, 'getCredentialsStatus');
```

Comment:
`checkIpOnly` doesn't check the user defined rate limit rules, so I don't think you want this?

Reply:
Thanks for this call out. I didn't realize custom config only applied to `customs.check`, so best to change it back. I tested both scenarios locally, and they both appeared to work. I was probably just over thinking this...